test/src/test/java/org/corfudb/runtime/view/AddressSpaceViewTest.java

```diff
@@ -84,6 +85,30 @@ public void ensureStripingWorks()
                 .isEmptyAtAddress(0);
     }
 
+    @Test
+    public void testUncachedWrites() {
```

Comment:
For completeness should we add a test that shows that regardless of the Cache Option if the cache is disabled no writes will be cached?

Reply:
Done.

Reply:
I can't see the case where we r.setCacheDisabled(true); did you push the change?

Reply:
Hmm, I think you are asking for the default write behavior (i.e. without passing a CacheOption, so I added : 

// write with the default write method
        r.getAddressSpaceView().write(new TokenResponse(2, epoch,
                        Collections.singletonMap(CorfuRuntime.getStreamID("stream1"), Address.NO_BACKPOINTER)),
                "payload".getBytes());

Reply:
I'll remove r.setCacheDisabled(false); because that's the point of this change, cache by-pass